### ISSUE TITLE
Word 2010: should be possible to get to header/footer with flat review or ObjectNav and NVDA+F within header should not report spelling error

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1943]
**Reported by kevinchao89 on 2011-11-21 02:26**
NVDA winwordTextInproc,4801
Word 2010

It's quite straight forward to insert header/footer and review it, provided one has not exited header/footer area. It would be great if it was possible to easily get to the header/footer using object navigation or flat review. I think ObjectNav would be great for this. Parent can be entire document, header, body, and footercan each be a child. THis would make it easier to read, review, and edit header/footer.

Within sample test doc, navigating to header, NVDA+F will report spelling error. where it should report this type of info in font info.

**Comment by bhavyashah** [comment]
The fact that we have moved on from flat review doesn't change anything with respect to this ticket, right? Regardless, in terms of priority assignments, I actually think a P3 may be appropriate because headers and footers are used very commonly in any word processor and the proposal sounds nice to have. Thoughts? @Qchristensen @feerrenrut @michaelDCurran

**Comment by michaelDCurran** [comment]
Implementing #7409 (move to uIA) will give   Word 365 this ability out of the box. I still do not believe this is possible in Word 2010. However, we'll accept a pr if someone comes up with something. No priority from NV Access's point of view.

**Comment by feerrenrut** [comment]
Given what @michaelDCurran has said, giving this a priority 4. We are happy to accept a PR to address this.

**Comment by josephsl** [comment]
Hi,

MOre than seven years later...

Now that we have a bit more access to Word via UIA, how feasible is a pull request to resolve this issue?

**Comment by Adriani90** [comment]
cc: @gerald-hartig, solving this would still improve the situation significantly. It is relevant for MS Office 365, both with and without UIA.

**Comment by gerald-hartig** [comment]
In Office 365 with UIA enabled you can get to the headers & footers via object navigation now.
